.github/workflows/release.yml

```diff
@@ -1,9 +1,12 @@
 name: Release CI
 
 on:
+  workflow_dispatch:
   push:
     branches:
     - master
+    - alpha
+    - beta
```

Comment:
Before this PR lands this file should show no changes. This will mean making a PR to current `master` updating this file to deploy from `master` and `22.x`.